examples/features/scale-from-zero/supplier-role.yaml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Let's move this into apps/nse-supplier-k8s/

Reply:
done